config/lib/jwtAuthentication.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var config = require('../config'),
+  jwt = require('jsonwebtoken');
+
+// build token auth service
+var tokenAuth = {
+  signToken: signAuthToken
+};
+
+// export the token auth service
+module.exports = tokenAuth;
```

Comment:
will do.  stole this from @mleanos PR.  Do you have an opinion on the file naming and location?  I am going to work on the tests and need to require the file so I want it to be settled on name and location.  See my comment on the main conversation